modules/peridynamics/src/materials/ComputeStrainBaseNOSPD.C

```diff
@@ -18,9 +18,14 @@ InputParameters
 ComputeStrainBaseNOSPD::validParams()
 {
   InputParameters params = MechanicsMaterialBasePD::validParams();
-  params.addClassDescription(
-      "Base class for Self-stabilized Non-Ordinary State-based PeriDynamic (SNOSPD) "
-      "correspondence material model");
+  params.addClassDescription("Base material strain class for peridynamic correspondence model");
+
+  MooseEnum stabilization_option("FORCE HORIZON");
+  params.addRequiredParam<MooseEnum>(
+      "stabilization",
+      stabilization_option,
+      "Available stabilization options for peridynamic correspondence model: " +
```

Comment:
The web documentation shows the options now, so we're going away from doing the printout of getRawNames() anymore. Just describe what the parameter does in that last argument to this function. Something like "Stabilization technique used for the peridynamic correspondence model"